lib/src/duration.dart

```diff
@@ -54,56 +54,56 @@ String prettyDuration(Duration duration,
     return sb.toString();
   };
 
-    final int weeks = (duration.inDays / 7).floor();
+  final int weeks = (duration.inDays / 7).floor();
```

Comment:
All this formatting is a result of running `dartfmt`, which should have been ran